### PR TITLE
Feature#13 Booking views and components to allow new flight registration

### PR DESCRIPTION
**Pull request by cortezroberto**
# Description
- **We've implemented CalendarPicker from 'react-native-calendar-picker' to handle flight date selection.**
- **We've implemented Picker from 'react-native-picker/picker' to handle place and passengers list selection.**
- **We've created the booking views to allow the user registering aflight:**
 - "From" view to allow the user to select start location.
 - "To" view to allow the user to select final location.
 - "Select date" view to allow the user to select date flight info.
 - "Passenger" view to allow the user to select how many passengers will fly.
 - "Results" view to show the information of the flight registered.

- **New components:**
- "ArrowBack" component to allow the user going back to previous screen.
- "Flight" icon component.
- "Calendar" component using CalendarPicker.
- "Place" and "Passeneger" components using Picker tha allows the user select using a list.

# Testing
- We've added a temporal link to test the views, the link is placed at the bottom of the "Sign up" view and it is named "Next".

### CODE REVIEW

**Review by martinmt-mx — APPROVED** [review]

File: FlightReservation/App.js [hunk 1]
```diff
@@ -20,6 +25,11 @@ const App = () => {
         <Stack.Navigator screenOptions={{header: () => null}}>
           <Stack.Screen name="SignUp" component={SignUp} />
           <Stack.Screen name="LogIn" component={LogIn} />
+          <Stack.Screen name="From" component={From} />
+          <Stack.Screen name="To" component={To} />
+          <Stack.Screen name="Date" component={Date} />
+          <Stack.Screen name="Passenger" component={Passenger} />
+          <Stack.Screen name="Results" component={Results} />
```
Comment (DemisRincon) [inline FlightReservation/App.js:32]: as a recommendation make an object array and then a map over that array

File: FlightReservation/src/components/booking/Passengers.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <SafeAreaView>
+      <View>
+        <Picker
+          selectedValue={passengers}
+          onValueChange={itemValue => setPassengers(itemValue)}>
+          <Picker.Item label="1" value="1" />
+          <Picker.Item label="2" value="2" />
+          <Picker.Item label="3" value="3" />
+          <Picker.Item label="4" value="4" />
+          <Picker.Item label="5" value="5" />
```
Comment (DemisRincon) [inline FlightReservation/src/components/booking/Passengers.js:17]: as a recommendation make an array and then a map over that array

File: FlightReservation/src/components/booking/Place.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <View>
+        <Picker
+          selectedValue={place}
+          onValueChange={itemValue => setSelectedPlace(itemValue)}>
+          <Picker.Item label="Select location:" />
+          <Picker.Item label="CDMX" value="Ciudad de Mexico" />
+          <Picker.Item label="AGS" value="Aguascalientes" />
+          <Picker.Item label="COL" value="Colima" />
+          <Picker.Item label="SIN" value="Sinaloa" />
+          <Picker.Item label="ZAC" value="Zacatecas" />
```
Comment (DemisRincon) [inline FlightReservation/src/components/booking/Place.js:18]: as a recommendation make an object array and then a map over that array

File: FlightReservation/src/screens/booking/Passenger.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <SafeAreaView>
+      <ArrowBack navigation={navigation} />
+      <View>
+        <Text>{route.params.fromPlace}</Text>
+        <Flight />
+        <Text>{route.params.toPlace}</Text>
+        <Text>Start date:</Text>
+        <Text>{startDate}</Text>
+        <Text>End date:</Text>
+        <Text>{endDate}</Text>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/booking/Passenger.js:30]: follow your pattern and use i18n, here is recommendable also use an array to that strings

File: FlightReservation/src/screens/booking/Results.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      <View>
+        <Text>{route.params.fromPlace}</Text>
+        <Flight />
+        <Text>{route.params.toPlace}</Text>
+        <Text>Start date:</Text>
+        <Text>{startDate}</Text>
+        <Text>End date:</Text>
+        <Text>{endDate}</Text>
+        <Text>{route.params.passengers}</Text>
+        <Text>Passengers</Text>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/booking/Results.js:30]: same as the prev recomendation

**Review by DemisRincon — CHANGES_REQUESTED** [review]
Check the comments

File: FlightReservation/App.js (hunk 1 above)
Comment (cortezroberto) [reply FlightReservation/App.js]: Done

File: FlightReservation/src/components/booking/Passengers.js (hunk 2 above)
Comment (cortezroberto) [reply FlightReservation/src/components/booking/Passengers.js]: Done

File: FlightReservation/src/components/booking/Place.js (hunk 3 above)
Comment (cortezroberto) [reply FlightReservation/src/components/booking/Place.js]: Done

File: FlightReservation/src/screens/booking/Passenger.js (hunk 4 above)
Comment (cortezroberto) [reply FlightReservation/src/screens/booking/Passenger.js:30]: Static text deleted

File: FlightReservation/src/screens/booking/Results.js (hunk 5 above)
Comment (cortezroberto) [reply FlightReservation/src/screens/booking/Results.js:30]: Static text deleted

**Review by DemisRincon — APPROVED** [review]

**Review by yovanha — APPROVED** [review]
Good!

**Review by hancel-avila — APPROVED** [review]
Nice work! 👍🏻

**Pull request merged**